test/CMakeLists.txt

```diff
@@ -48,14 +48,55 @@ add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/small
 add_custom_target(symlink_libsmall_test_binaries ALL
```

Comment:
> Note, tests for 'small' lib should be considered as unit tests as well,
but for them we need to do the corresponding changes in tarantool/small
submodule and after that add building test binaries to `unit-test-bin`
target. It will be implemented in a separate PR when changes will be
done in tarantool/small submodule.

I'm not sure that I understand you here. Is there an issue on this topic? It would be good to file it (if there is no one) and link from here.

Reply:
There is no issue for this. I planned to create it later, but now I am not sure that I will be doing it at all. From your comment above I realised that we can build unit tests and small tests in such a way:

```bash
cmake .
make -j -C test/unit
make -j -C test/small
```
and run test-run by `cd test && ./test-run.py unit/ small/`.

For building all tests (unit, small, and functional):

```bash
cmake .
make -j -C test
```

So I would revert all my changes for CMakeLists.txt files and left this as it is now because we have a way to build tests without any changes in CMakeLists.txt files.